src/impl_methods.rs

```diff
@@ -1707,6 +1707,36 @@ where
         unsafe { Some(ArrayView::new(self.ptr, dim, broadcast_strides)) }
     }
 
+    /// Calculate the views of two ArrayBases after broadcasting each other, if possible.
```

Comment:
It will be an internal method so it doesn't matter so much, but we don't use this terminology ("ArrayBases") in the docs.
```suggestion
    /// For two arrays or views, find their common shape if possible and broadcast them as array views into that shape
```